pkg/test/packageTest.targets

```diff
@@ -101,7 +101,7 @@
   <Target Name="_test"
           DependsOnTargets="$(TestDependsOn)">
     <MakeDir Directories="$(IntermediateOutputPath)" />
-    <Touch AlwaysCreate="true" Files="$(IntermediateOutputPath)\.testComplete" />
+    <Touch AlwaysCreate="true" Files="$(IntermediateOutputPath)\$(RuntimeIdentifier)\.testComplete" />
```

Comment:
Note below at line 116 I was expecting this file to be written but didn't write it. ☹️